recipes/wpewebkit.recipe

```diff
@@ -30,25 +28,23 @@ class Recipe(recipe.Recipe):
         'openjpeg',
     ]
     platform_deps = {Platform.ANDROID: ['gnustl']}
-    # TODO: support accessibility, gstreamer-gl and woff2
+    # TODO: support accessibility woff2
     configure_options = '-DPORT=WPE \
                         -DENABLE_ACCESSIBILITY=OFF \
                         -DUSE_GSTREAMER_GL=ON \
                         -DUSE_GSTREAMER_NATIVE_VIDEO=ON \
                         -DUSE_GSTREAMER_NATIVE_AUDIO=ON \
                         -DUSE_WOFF2=OFF \
-                        -DUSE_SYSTEMD=OFF \
+                        -DENABLE_DOCUMENTATION=OFF \
+                        -DENABLE_INTROSPECTION=OFF \
+                        -DENABLE_JOURNALD_LOG=OFF \
                         -DENABLE_BUBBLEWRAP_SANDBOX=OFF \
-                        -DWPE_ENABLE_PROCESS=1 \
```

Comment:
why did you remove this? Now patch 0006-Enable-libwpe-process-management-api.patch has no effect anymore

Reply:
I didn't want to remove it, this is an error. This is why it is not working with wpe-android. I fixed it :+1: